update readme with zip-packs command (#775)
demisto/content-docs: docs/concepts/demisto-sdk.md

For additional information see [here](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/upload/README.md#upload).

### zip-packs
Use this command to generate zipped packs and upload them to the Marketplace in Cortex XSOAR. Be sure to set up the `DEMISTO_BASE_URL` 
and the `DEMISTO_API_KEY` prior to running this command in order to establish a connection between `demisto-sdk` and the XSOAR server.

#### Examples and Use Cases:
 - Zip a pack:
    ```buildoutcfg

>     demisto-sdk zip-packs -i Campaign -o someDir/zippedPacks

Reviewer: ```suggestion
    demisto-sdk zip-packs -i Campaign -o Packs/Campaign
```

    ```

>    This will zip the `Campaign` pack into Campaign.zip file in the `DestinationDir` directory.

Reviewer: in the rest of the docs we give file paths that really look like file paths, like: 
`Packs/myNewPack/Integrations` found in https://xsoar.pan.dev/docs/concepts/demisto-sdk#init

I think it looks better, lets try and change all `DestinationDir` to something of that sort

>    This will zip the `Campaign` pack into Campaign.zip file in the `someDir/zippedPacks` directory.

Reviewer: ```suggestion
   This will zip the `Campaign` pack into Campaign.zip file in the `Packs/Campaign` directory.
```

   
   
 - Zip and upload a whole pack to the Marketplace:
    ```buildoutcfg

>     demisto-sdk zip-packs -i Campaign -o someDir/zippedPacks -u

Reviewer: ```suggestion
    demisto-sdk zip-packs -i Campaign -o Packs/Campaign -u
```
